ush/logger.py

```diff
@@ -0,0 +1,152 @@
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
```

Comment:
This is pulled directly from eva I'm assuming? I think Rahul has a logger that is similar in wxflow that we can probably leverage.